src/brogue/MainMenu.c

```diff
@@ -46,15 +46,27 @@ void drawMenuFlames(signed short flames[COLS][(ROWS + MENU_FLAME_ROW_PADDING)][3
     color tempColor = {0};
     const color *maskColor = &black;
     char dchar;
+#ifdef UNSTABLE_VERSION_STRING
+    short unstableVersionStringLength;
+#endif
 
     versionStringLength = strLenWithoutEscapes(BROGUE_VERSION_STRING);
+#ifdef UNSTABLE_VERSION_STRING
+    unstableVersionStringLength = strLenWithoutEscapes(UNSTABLE_VERSION_STRING);
+#endif
 
     for (j=0; j<ROWS; j++) {
         for (i=0; i<COLS; i++) {
-            if (j == ROWS - 1 && i >= COLS - versionStringLength) {
-                dchar = BROGUE_VERSION_STRING[i - (COLS - versionStringLength)];
-            } else {
-                dchar = ' ';
+            dchar = ' ';
+            if (j == ROWS - 1) {
+#ifdef UNSTABLE_VERSION_STRING
+                if (i < unstableVersionStringLength) {
+                    dchar = UNSTABLE_VERSION_STRING[i];
```

Comment:
Might be easier to append to BROGUE_VERSION_STRING, it's only cosmetic

Reply:
`BROGUE_VERSION_STRING` is what gets written in `broguerec` files, isn't it?  I need to go back and look but I seem to remember deliberately not touching that in order to not break play backs.

Reply:
Oh, I see.  `BROGUE_RECORDING_VERSION_STRING` is used in recording files.  We could safely add the unstable component to `BROGUE_VERSION_STRING`.

I'm happy either keeping the constant broken out, or with merging them.  What do you think?

Reply:
Actually merging them looks pretty safe and clean.  I'll just do that.